Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN yum install -y -q http://pkg.jenkins-ci.org/redhat/jenkins-${JENKINS_VERSION
 
 ADD docker.repo /etc/yum.repos.d/docker.repo
 
-ENV DOCKER_VERSION 1.11.0
+ENV DOCKER_VERSION 1.9.1
```

Comment:
I think we're actually using 1.10.1.  We only really like software that cuts you.